doc/api/net.md

```diff
@@ -658,9 +658,9 @@ called with `{port: port, host: host}` as `options`.
 added: v6.1.0
 -->
 
-If `true` -
+If `true`,
 [`socket.connect(options[, connectListener])`][`socket.connect(options)`]
-was called and haven't yet finished. Will be set to `false` before emitting
+was called and has not yet finished. Will be set to `true` before emitting
```

Comment:
Micro-nit: While we're in here, get rid of `yet`?

For some other PR perhaps: What does "finished" mean in this context? The connection has not been established? I imagine it doesn't mean that the method is still executing but that is what a naive interpretation might be....